docs/source/slack.rst

```diff
@@ -72,6 +72,22 @@ Do not try to assume which pronoun to use while talking to someone. It is much
 nicer if you use gender neutral words (or the names) in any type of
 communication. This guide has more details which one should read.
 
+Adjust notification time for Slack
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, Slack turns off notifications after 10 PM.
+To avoid missing out on any notifications after that, extend the default
+notification time. Reach out the *Do Not Disturb* icon, and set your
+notification time frame.
+
+Try out the dark theme
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Slack offers a dark theme as well.
```

Comment:
Slack offers a dark theme as well, which is much more pleasant and comfortable for your eyes. Tu turn it on, head over to ...

Reply:
> Tu turn it on
*Aap